prow/oss/cluster/deck_private_deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: deck-private
+  namespace: default
```

Comment:
I'm not sure why this defaulted to `test-pods`, but it should be explicit anyways.